lavamoat/webpack/policy-override.json

```diff
@@ -0,0 +1,328 @@
+{
+  "resources": {
+    "bootstrap": {
+      "globals": {
+        "document": true,
+        "window": true,
+        "Element": true,
+        "HTMLElement": true,
+        "Node": true,
+        "Event": true,
+        "CustomEvent": true,
+        "document.body": true,
+        "document.documentElement": true,
+        "document.createElement": true,
+        "document.createEvent": true,
+        "document.querySelector": true,
+        "document.querySelectorAll": true,
+        "document.addEventListener": true,
+        "document.removeEventListener": true,
+        "window.addEventListener": true,
+        "window.removeEventListener": true,
+        "window.setTimeout": true,
+        "window.clearTimeout": true,
+        "window.setInterval": true,
+        "window.clearInterval": true,
+        "window.requestAnimationFrame": true,
+        "window.cancelAnimationFrame": true,
+        "window.getComputedStyle": true,
+        "window.Event": true,
+        "window.CustomEvent": true,
+        "window.MutationObserver": true,
+        "MutationObserver": true
+      },
+      "packages": {
+        "jquery": true,
+        "popper.js": true
+      }
+    },
+    "popper.js": {
+      "globals": {
+        "document": true,
+        "window": true,
+        "Element": true,
+        "HTMLElement": true,
+        "getComputedStyle": true,
+        "window.getComputedStyle": true,
+        "requestAnimationFrame": true,
+        "cancelAnimationFrame": true
+      }
+    },
+    "jquery": {
+      "globals": {
+        "document": true,
+        "window": true,
```

Comment:
When you allow `window`, does it allow all `window.*`?

Reply:
Not really, lavamoat follows the principle of least privilege, so setting `window: true` only gives basic access to the window object itself, so you do these:

```
if (window) {
...
}
```

`typeof window`

But you need specific permissions for window's methods, e.g.:

// These need explicit permissions like "window.localStorage": true
window.localStorage.getItem('key')
window.sessionStorage.setItem('key', 'value')

// These need "window.location": true
window.location.href
window.location.reload()

// These need "window.history": true
window.history.pushState()
window.history.back()

// These need "window.alert": true
window.alert('message')